source/Sashimi.AzureServiceFabric/AzureServiceFabricAppHealthCheckActionHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Sashimi.Server.Contracts.ActionHandlers;
+using Sashimi.Server.Contracts.Calamari;
+
+namespace Sashimi.AzureServiceFabric
+{
+    class AzureServiceFabricAppHealthCheckActionHandler : IActionHandlerWithAccount
```

Comment:
Does this class also need to be registered in the autofac module? I see you've done that for WebApp and CloudService

Reply:
Fixed